Cargo.toml

```diff
@@ -10,3 +10,5 @@ readme = "README.md"
 edition = "2018"
 
 [dependencies]
+async-trait = "0.1.24"
+futures = "0.3.4"
```

Comment:
Compile times are generally better if only including `futures-io` for the AsyncRead + AsyncWrite traits. Alternatively `async-std` provides a `core` flag iirc.